packages/google-cloud-serverless/src/utils.ts

```diff
@@ -17,14 +17,14 @@ export function domainify<A extends unknown[], R>(fn: (...args: A) => R): (...ar
  * @returns wrapped function
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function proxyFunction<A extends any[], R, F extends (...args: A) => R>(
+export function proxyFunction<F extends (...args: any[]) => unknown>(
```

Comment:
sadly this somehow still has to be `any`, not quite sure why but didn't feel like investing more time into this...